webpack/ForemanInventoryUpload/Components/InventoryFilter/InventoryFilterReducer.js

```diff
@@ -28,15 +29,15 @@ export default (
       });
     case LAYOUT_CHANGE_ORG: {
       const { title } = org;
-      const term = title === ANY_ORGANIZATION ? '' : title;
+      const term = title === __(ANY_ORGANIZATION) ? '' : title;
```

Comment:
Does the `org` object has a `value` that doesn't get translated?

Reply:
unfortunately it only has `title` key